source/crc32.cpp

```diff
@@ -57,36 +57,39 @@ static const unsigned long crcTable[256] = {
     0x2d02ef8dUL
 };
 
-unsigned long crc32FromUpcaseString(const char *str)
-{
-    unsigned long crc = 0xFFFFFFFF;
-    while (*str)
-        crc = crcTable[(crc^toupper(*str++)) & 0xff] ^ (crc >> 8);
-    return crc;
-}
 
-unsigned long crc32FromUpcaseStdString(const std::string& str)
-{
-    return crc32FromUpcaseString(str.c_str());
-}
+unsigned long crc32FromUpcaseString(const char *str) {
```

Comment:
Majority of the project uses bracket in new line style. Please keep it that way.